src/helpers/timeFormattedToMultipleZones.js

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from 'luxon'
+import _ from 'underscore'
+
+// PST https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+// India's IANA zone https://en.wikipedia.org/wiki/Time_in_India
+const zones = [
+  { shortname: 'PT', ianaZone: 'America/Los_Angeles' },
+  { shortname: 'IST', ianaZone: 'Asia/Kolkata' },
+  { shortname: 'GMT', ianaZone: 'Etc/GMT' }
```

Comment:
Any particular reason we are calling this GMT instead of UTC? UTC seems to be the most commonly used name across Azure.

Reply:
Good catch.